Modules/CustomerHistory/Frames.lua

```diff
@@ -24,15 +24,46 @@ function CraftSim.CUSTOMER_HISTORY.FRAMES:Init()
     local function createContent(frame)
         self.frame:Hide()
         self.frame.content.customerDropdown = CraftSim.GGUI.Dropdown({
-            parent=self.frame.content, anchorParent=self.frame.title.frame, anchorA="TOP", anchorB="TOP", offsetY=-30, width=170, offsetX=-170,
+            parent=self.frame.content, anchorParent=self.frame.title.frame, anchorA="TOP", anchorB="TOP", width=170, offsetX=-170, offsetY=-30,
             initialValue=nil,
             initialLabel="",
             label=CraftSim.LOCAL:GetText(CraftSim.CONST.TEXT.CUSTOMER_HISTORY_DROPDOWN_LABEL),
             initialData={},
             clickCallback=function (_, _, item) CraftSim.CUSTOMER_HISTORY.FRAMES:SetCustomer(item) end
         })
+        self.frame.content.deleteButton = CraftSim.GGUI.Button({
+            parent=frame.content,anchorParent=self.frame.content.customerDropdown.frame, anchorA="LEFT", anchorB="RIGHT", offsetX=0, offsetY=2,
+            label=CraftSim.LOCAL:GetText(CraftSim.CONST.TEXT.CUSTOMER_HISTORY_DELETE_BUTTON), sizeX=15, sizeY=20, adjustWidth=true,
+            clickCallback=function ()
+                CraftSim.CUSTOMER_HISTORY.db.realm[CraftSim.CUSTOMER_HISTORY.db.realm.lastCustomer] = nil
+                self:ResetDropdown()
+                self.frame.content.messageBox:Clear()
+                self.frame.content.totalTip:SetText(string.format("%s%s", CraftSim.LOCAL:GetText(CraftSim.CONST.TEXT.CUSTOMER_HISTORY_TOTAL_TIP), CraftSim.GUTIL:FormatMoney(0)))
```

Comment:
It would make more sense to call `self:LoadTotalTip` here, by adding some logic if `customer` is `nil` instead of doing this.